NitroxClient/Communication/Packets/Processors/ChatMessageProcessor.cs

```diff
@@ -19,14 +19,20 @@ public ChatMessageProcessor(PlayerManager remotePlayerManager, PlayerChatManager
 
         public override void Process(ChatMessage message)
         {
-            Optional<RemotePlayer> remotePlayer = remotePlayerManager.Find(message.PlayerId);
-
-            if (remotePlayer.IsPresent())
+            if (message.PlayerId != ushort.MaxValue)
```

Comment:
Let's add a constant in ChatMessage for the server command id.  So we can easily reference it and understand the code.